docs/ref/models/expressions.txt

```diff
@@ -289,7 +289,11 @@ The ``Func`` API is as follows:
 
         The ``as_vendor()`` methods should use the ``function``, ``template``,
         ``arg_joiner``, and any other ``**extra_context`` parameters to
-        customize the SQL as needed. For example:
+        customize the SQL as needed. ``extra_context`` must not contain
+        untrusted user input as these values aren't escaped when they're
```

Comment:
I sort of agree with @hynekcer . I'd replace "these values aren't escaped" with something like "these values are interpolated into the SQL string, not passed as parameters, and aren't even escaped".